vignettes/ChromBand.Rmd

```diff
@@ -0,0 +1,379 @@
+  y_i = \beta_0 + \beta_1 a_{ij_1} + \beta_2 a_{ij_2} + \varepsilon_{i}$$ and testing the null hypothesis $H_0: \beta_1 = 0$. Multiple significant sub-bands and multiple levels of nesting can be incorporated by including further terms in the model. The complete process can be summarized as follows: Start by marginally testing each band which has no sub-bands. For all other bands, first test all sub-bands, then test the current band using a linear model that includes a term for each *significant* sub-band.
+
+We apply this procedure to perform global tests using per-gene $t$-statistics as a measure of differential expression in BCR/ABL relative to NEG samples. As with the Hypergeometric tests, we start by creating objects of class *"ChrMapLinearMParams"*.
+
+```{r LMtestSetup}
+
+params <- new("ChrMapLinearMParams",
+              conditional = FALSE,
+              testDirection = "up",
+              universeGeneIds = entrezUniverse,
+              geneStats = tstats,
+              annotation = "hgu95av2",
+              pvalueCutoff = 0.01, 
+              minSize = 4L)
+
+params@graph <- makeChrBandGraph(params@annotation, params@universeGeneIds)
+params@gsc <- makeChrBandGSC(params@graph)
+paramsCond <- params
+paramsCond@conditional <- TRUE
+```
+
+The tests are performed, and the results summarized by
+
+```{r LMtest}
+lmans <- linearMTest(params)
+lmansCond <- linearMTest(paramsCond)
+chrSortOrder(summary(lmans))
+
+chrSortOrder(summary(lmansCond))
+##
+```
+
+<figure id="fig:dotplot1p362">
+
+```{r echo=FALSE}
+tmpSet <- subsetByBand(nsFiltered, ct, "1p36.2")
+esetBWPlot(tmpSet, ylab="1p36.2", layout = c(2, 8),
+           par.strip.text = list(cex = 0.8))
+```
+
+<figcaption>Figure 3: Expression values for the BCR/ABL and NEG samples for the genes located within 1p36.2, which is declared as significant by the Hypergeometric test, but not by GSEA. Genes in the selected list are highlighted with a shaded background. For these genes, The NEG samples, those with no known cytogenetic abnormalities, have significantly lower expression than the BCR/ABL samples. However, the direction is reversed (albeit mildly) for many of the other genes.</figcaption>
+</figure>
+
+These examples only test for consistently upregulated categories; similar calls with `testDirection = "down"` can be used to test for downregulation. As we see, the GSEA approach picks out many more bands as significant, but there is some concordance with the Hypergeometric approach. For example, 7q31, 8p22, and 14q22.2 come up in both analyses. <span><a href="#fig:dotplot1p362"> Figure 3</a><span> shows box-and-whisker plots of genes in one category (1p36.2) that is declared as significant by the Hypergeometric test, but not by GSEA. It is produced by
```

Comment:
Remove `<span>` tags.

Reply:
That is the **Html syntax to keep an inline anchor, else it all going to be rendered as broken lines, please check all the documentations you gave us to read.**